src/pages/home/components/LandingImage.js

```diff
@@ -16,15 +18,22 @@ const useStyles = makeStyles({
 
 export const LandingImage = () => {
   const classes = useStyles();
+  const isMobileView = useMediaQuery({
+    query: "(max-width: 639px)",
+  });
 
   return (
     <div className={`h-screen justify-center items-center`}>
       <div className={`h-screen absolute w-full ${classes.bgImage}`}></div>
       <div className="relative flex justify-center items-center h-full">
         <div className="w-full mt-1/10">
-          <img src={Gisau} alt="Gisau Logo" className="w-1/5 m-auto"></img>
-          <h1 className="text-white my-8 font-proxima-nova text-center text-[2.5rem] font-semibold leading-12">
-            Gado-Gado Indonesian Student Association <br />
+          <img
+            src={Gisau}
+            alt="Gisau Logo"
+            className="w-[30%] md:w-1/4 lg:w-1/5 m-auto"
+          ></img>
+          <h1 className="text-white my-4 md:my-8 mx-2 font-proxima-nova text-center font-semibold text-lg sm:text-2xl md:text-4xl lg:text-[2.5rem] lg:leading-12">
+            Gado-Gado Indonesian Student Association {!isMobileView && <br />}{" "}
```

Comment:
> What is `{" "}` for?

Because for screen size 639px and below, I don't want a new line for the "of UBC". So it is to make sure there is a space in between when it becomes one line.

Reply:
> Also you don't need to use `<h1>` tag if you end up overwriting the font size and the line height. Try using `<div>` or `<span>`

I use `<h1>` just to easily spot that the text is a title. Any reason why `<span>` or `<div>` is better?

Reply:
> > What is `{" "}` for?
> 
> Because for screen size 639px and below, I don't want a new line for the "of UBC". So it is to make sure there is a space in between when it becomes one line.

but after Association there's already an extra whitespace, so it'll be Association*space**space*of UBC. 

<br>
<br>
<br>

> I use `<h1>` just to easily spot that the text is a title. Any reason why `<span>` or `<div>` is better?

`<span>`/`<div>` has no built-in styling, that's it. Yeah you can still use <h1> then

Reply:
> > > What is `{" "}` for?
> > 
> > 
> > Because for screen size 639px and below, I don't want a new line for the "of UBC". So it is to make sure there is a space in between when it becomes one line.
> 
> but after Association there's already an extra whitespace, so it'll be Association_space**space_of UBC.
> 
> 
> 
> > I use `<h1>` just to easily spot that the text is a title. Any reason why `<span>` or `<div>` is better?
> 
> `<span>`/`<div>` has no built-in styling, that's it. Yeah you can still use `<h1>` then

It actually doesn't effect anything. If I remove it, it doesn't change anything.